src/Compilers/CSharp/Portable/Binder/BinderFlags.cs

```diff
@@ -113,9 +113,9 @@ internal enum BinderFlags : uint
         InEEMethodBinder = 1 << 30,
 
         /// <summary>
-        /// Assume '#nullable disable' context.
+        /// Skip binding type arguments in type constraints
         /// </summary>
-        IgnoreNullableContext = 1u << 31,
+        LightweightTypeConstraintBinding = 1u << 31,
```

Comment:
>LightweightTypeConstraintBinding [](start = 8, length = 32)

It looks like this flag is used in a function that is not specific to constraint binding, but rather it is our core code path to bind types in general. Let's keep the name of this flag indicating what that function should do, instead of reflecting mainly why. I suggest "SuppressTypeArgumentBinding", or something like that.